ompi/mca/coll/han/coll_han_topo.c

```diff
@@ -0,0 +1,361 @@
+/*
+ * Copyright (c) 2004-2007 The Trustees of Indiana University and Indiana
```

Comment:
Less copyright

Reply:
Same as coll_han_subcomm.c : 
This file is also a enhancement (+split +renaming) of https://github.com/bosilca/ompi/blob/future_adapt/ompi/mca/coll/future/coll_future_module.c that already contained these copyrights

Reply:
same answer as above.

Reply:
Fixed in a746a13541eb11a76663d06be244011c8de2672c